src/ROUTES.ts

```diff
@@ -89,6 +89,7 @@ const ROUTES = {
     SETTINGS_PRONOUNS: 'settings/profile/pronouns',
     SETTINGS_PREFERENCES: 'settings/preferences',
     SETTINGS_SUBSCRIPTION: 'settings/subscription',
+    SETTINGS_SUBSCRIPTIONS_SIZE: 'settings/subscription-size',
```

Comment:
we have different address for this - settings/subscription/subscription-size

Reply:
yeah, mb, fixing it!